docs/lib/auth/menu.json

```diff
@@ -11,14 +11,14 @@
     "switch-auth",
     "customui",
     "advanced",
-
-    "how-it-works",
-		"guest-access",
-		"working-with-api",
-		"drop-in-auth",
-		"federated-identities",
-		"hosted-ui",
-		"custom-auth-flow",
-		"device-features"
+    
+    "register-user",
+    "authenticate-user",
+    "auth-events",
+    "working-with-api",
+    "user-attributes",
+    "device-features",
```

Comment:
I think "Remember device" would be clearer than "Device features"

Reply:
Hmm, would it make sense to have a section for "Password Management" where we cover updating it, resetting it, confirming the reset?